sdk/core/azure-core/src/operation_status.cpp

```diff
@@ -11,4 +11,6 @@ namespace Azure { namespace Core {
   const OperationStatus OperationStatus::Failed{"Failed"};
   const OperationStatus OperationStatus::Cancelled{"Cancelled"};
 
+  // LCOV_EXCL_START
 }} // namespace Azure::Core
+// LCOV_EXCL_STOP
```

Comment:
How would we know when to do these type of exclusion? I am not sure if this is a good idea. Given anything over 95% starts to have diminishing returns in terms of effort, could we leave out these single-line "lack of coverage" as part of that 5%, and not have such annotations?

Reply:
There is no way to know in advance. The idea was that you look at each file, and if you see a noise, you mark it with annotations, indicating that it has been reviewed and nothing can be done.